### ISSUE TITLE
System freeze after running rclone vfs/refresh recursive=true fast-list

### ISSUE THREAD

**Issue by MythodeaLoL** [javi11/usenet-drive#114]
UsenetDrive doesn't seem to handle rclone's metadata scanning very well.

when executing the command `rclone rc vfs/refresh recursive=true _async=true --timeout 60m --rc-addr localhost:5572 --rc-user user --rc-pass pass --fast-list -v`
the system feeze, and UsenetDrive reports that there were not enough resources to complete the operation, it does not read the file `"level":"ERROR","msg":"Error getting metadata for file ... .nzb`

rclone pre-caches some file information, downloading and saving it in the cache.

**After it crashes, it takes a few minutes to come back, I'm running on a Ryzen 5 and NVMe.**

I really don't know the cause, I tested several rclone commands exhaustively, basic configurations, minimum configurations, and optimized ones, and I always get this problem. The system locks and takes a few minutes to unlock.

The settings used were:
```
--vfs-cache-mode full ^
--dir-cache-time 96h ^
--vfs-cache-max-age 48h ^
--vfs-read-chunk-size 20M ^
--vfs-read-chunk-size-limit 1G ^
--buffer-size 1M ^
--network-mode ^
```

**Comment by javi11** [comment]
Hey @MythodeaLoL could you share the logs, if there is an error reading metadata is because the nzb do not have the needed information. It is true that it should not block your pc

**Comment by javi11** [comment]
Okey, I see that the error is ¨There are not enough system resources to complete the requested service.¨ which is only reproducible on windows systems. I think in order to fix this I have to implement a kind of buffer on reading the nzb file. The tool it was never tested on windows and is therefore not 100% compatible yet. I have other issues first to do before this one. Anyway, thanks for reporting it :). 
In the mean time, I recommend you to use the docker version of it.

**Comment by MythodeaLoL** [comment]
**Update:**

I tested it in an isolated environment on VMWare, and it apparently worked without any problems.

After that, I tested again and everything seems to have worked perfectly, I suspect that Kaspersky may have interfered in some way, I added all possible exclusions, both for UsenetDrive and Rclone, and for the Google Drive app.

For now I'm not experiencing any crashes or errors and everything works very quickly.

**I'm closing the issue for now.**
